renovate.json5

```diff
@@ -20,6 +20,9 @@
       groupName: 'devDependencies',
       // But exclude some specific packages from this group
       excludePackageNames: ['typescript'],
+      lockFileMaintenance: {
```

Comment:
I am not sure if lockFileMaintenance can be used for specific packages. From what I remember, I think it just deletes the lock file and regenerates it so all packages with semver ranges use the latest.

Reply:
afaik, i think it can be used for specific packages, I saw a bunch of examples on github of other people using it this way -> https://github.com/search?q=lockFileMaintenance&type=code

Reply:
interesting, wonder how this works. Does it upgrade all sub dependencies of matching dependencies on the lock files? It sounds like an expensive operation to do, but we can find out.